go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/hashicorp/terraform-exec
 
-go 1.18
+go 1.20
```

Comment:
This is the least version supporting `Cancel` and ` WaitDelay`: https://tip.golang.org/doc/go1.20#minor_library_changes